ballet/validation/common.py

```diff
@@ -105,10 +105,12 @@ def __init__(self, project, differ=None):
 
         if self.differ is None:
             pr_num = self.project.pr_num
+            repo = self.project.repo
+            if pr_num is None:
```

Comment:
incorporate this into the if/else directly. otherwise, LMBD created and overwritten by TPRBD in CI environment

Reply:
thanks! this was causing bugs earlier, I think the merge did something to revert it b/c it was made into a new file.